src/helpers/JWT.js

```diff
@@ -3,15 +3,21 @@ import { SECRET_KEY, EXPIRATION_DURATION } from '../config/constants';
 /**
  * Class handles JWT authentication
  */
-class JWT {
+export default {
   /**
-   * Method to generate token from userId and role
+   * Method to generate token
    * @param {object} data
    * @returns {string} generated token
    */
-  static async generateToken(data) {
+  generateToken: async (data) => {
```

Comment:
Just a suggestion why not just export the function instead of putting it within an object again